src/python/WMCore/Storage/StageOutMgr.py

```diff
@@ -16,15 +16,27 @@
 import WMCore.Storage.Backends
 import WMCore.Storage.Plugins
 
-from WMCore.Services.Dashboard.DashboardAPI import stageoutPolicyReport
 from WMCore.Storage.DeleteMgr import DeleteMgr
 from WMCore.Storage.Registry import retrieveStageOutImpl
 from WMCore.Storage.StageOutError import StageOutFailure
 from WMCore.Storage.StageOutError import StageOutInitError
 from WMCore.WMException import WMException
 
 
-# If we don't import them, they cannot be ever used (bad PyCharm!)
+def stageoutPolicyReport(fileToStage, pnn, command, stageOutType, stageOutExit):
+    """
+    Prepare report about stageout policies (probably only used for the old SSB Dashboard).
+    It reports whether the stage out was made local/fallback/direct and/or other possible
+    issues related to prod/analysis jobs.
+    """
+    tempDict = {}
+    tempDict['LFN'] = fileToStage['LFN'] if 'LFN' in fileToStage else None
+    tempDict['PNN'] = pnn if pnn else fileToStage['PNN'] if 'PNN' in fileToStage else None
```

Comment:
The logic seems correct, but I find this one hard to read.

Reply:
Agreed! Another copy/pasting from a module that is being removed. I can update it though.

Reply:
no worries for updating it right now ... we've been living with this since ever obviously.